index.js

```diff
@@ -73,3 +101,19 @@ window.onload = () => {
   }
   library.saver();
 };
+
+function displayDate() {
+  const now = document.getElementById('date');
+  const date = Date.now();
+  const options = {
+    weekday: 'long',
+    year: 'numeric',
+    month: 'long',
+    day: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+  };
+  now.innerHTML = new Intl.DateTimeFormat('en-US', options).format(date);
+}
```

Comment:
- [ ] Please remember that you are supposed to use the `Luxon` library to get and display the current date, here you are using `Date.now()` to get the date, kindly fix this 😄